cves/kernel/CVE-2015-6526.yml

```diff
@@ -167,8 +181,13 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    This vulnerability involves understanding the context in which the function
+    causing the vulner is used. To test this feature, the automated tool would
+    have to create a program with a malicious stack in order for the Linux perf
+    tool to crash. This may be possible with the use of mocking, but testing
+    would still require specific knowledge.
```

Comment:
This sounds good—I recommend explaining what "mocking" is.

Reply:
Good idea!